src/imports.rs

```diff
@@ -159,8 +159,9 @@ impl Rewrite for ast::ViewPath {
 
 impl<'a> FmtVisitor<'a> {
     pub fn format_imports(&mut self, use_items: &[ptr::P<ast::Item>]) {
-        let mut last_pos =
-            use_items.first().map(|p_i| p_i.span.lo - BytePos(1)).unwrap_or(self.last_pos);
+        let mut last_pos = use_items.first()
+            .and_then(|p_i| p_i.span.lo.0.checked_sub(1).map(|span_lo| BytePos(span_lo)))
```

Comment:
Can the `map` go outside the `and_then`? So we have `.first().and_then(...).map(...).unwrap_or(...)`